templates/travis/.travis/publish_client_gem.sh.j2

```diff
@@ -17,15 +17,17 @@ export DESCRIPTION="$(git describe --all --exact-match `git rev-parse HEAD`)"
 if [[ $DESCRIPTION == 'tags/'$REPORTED_VERSION ]]; then
   export VERSION=${REPORTED_VERSION}
 else
+  # Daily publishing of development version (ends in ".dev")
+  [ "${REPORTED_VERSION%.dev}" != "${REPORTED_VERSION}" ] || exit 1
```

Comment:
The test condition exits the entire script if the string does not end in .dev, correct?

And the test condition should never be reached for tagged releases versions, because of the original "if" condition?

Reply:
That is the plan, but i have it in place for debian, and it does not work as expected. I can just remove this check, and we only add the additional dot. That would at the very least give us versions that grow in the right order.
The original problem is, that when this script runs daily and the version is reported as `1.2.3rc5` it will published as `1.2.3.rc5201911111111` which will be bigger than `1.2.3.rc6`. The dot alone should help there. Also any non digit should to.

Reply:
I had to remove the dot again, as python really did not like it.